pkg/controller/dnsendpoint/dnsendpoint_controller.go

```diff
@@ -2,12 +2,14 @@ package dnsendpoint
 
 import (
 	"context"
+	"fmt"
```

Comment:
Is there a better name for this controller now?

Reply:
Yes, there is a better name. It may even be better to just include this functionality in the DNSZone controller. However, I wanted to limit the changes for this PR. I do plan on, at least, renamed this controller in future work.